### PR TITLE
feat: simplified duration dropdown

### PR DESCRIPTION
**Pull request by cbachmeier**
Simplified duration dropdown on list page

Also fixes bug where bottom padding in sort dropdown was the wrong color

### CODE REVIEW

File: src/nft/components/profile/list/SetDurationModal.tsx [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+            mini
+            miniPrompt={displayDuration + (displayDuration === duration ? pluralize(amount) : 's')}
+            left={38}
+            top={38}
+          />
+        </DropdownWrapper>
+      </InputWrapper>
+      {errorState !== ErrorState.valid && (
+        <ErrorMessage className={caption}>
+          {' '}
```
Comment (grabbou) [inline src/nft/components/profile/list/SetDurationModal.tsx:129]: What's the reason for this character?

File: src/nft/components/profile/list/SetDurationModal.tsx [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+          <WarningIcon /> {errorState === ErrorState.overMax ? 'Maximum 6 months' : 'Set duration'}
```
Comment (grabbou) [inline src/nft/components/profile/list/SetDurationModal.tsx:130]: nit: moving to `getMessageForError` + a switch/case for future extendability? (if any)

File: src/nft/components/profile/list/SetDurationModal.tsx [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+        </ErrorMessage>
+      )}
+    </ModalWrapper>
+  )
+}
+
+const convertDurationToExpiration = (amount: number, duration: Duration) => {
+  const durationFactor =
+    duration === Duration.hour ? 1 : duration === Duration.day ? 24 : duration === Duration.week ? 24 * 7 : 24 * 30
```
Comment (grabbou) [inline src/nft/components/profile/list/SetDurationModal.tsx:139]: For this kind of nested tenary, I always tend to prefer a `switch(true) case "expression"`, but it's just personal preference, so leaving as a nit for you to consider

File: src/nft/components/profile/list/SetDurationModal.tsx [hunk 4]
```diff
@@ -0,0 +1,141 @@
...
+      {
+        displayText: 'Months',
+        onClick: () => selectDuration(Duration.month),
+      },
+    ],
+    []
+  )
+  useEffect(() => {
+    const expiration = convertDurationToExpiration(amount, duration)
+    if (expiration * 1000 - Date.now() < ms`60 seconds`) setErrorState(ErrorState.empty)
```
Comment (grabbou) [inline src/nft/components/profile/list/SetDurationModal.tsx:95]: Question: do we maintain a general guideline as far as if statements? Shall we skip `{}`? If yes, when?

**Review by grabbou — APPROVED** [review]

File: src/nft/components/profile/list/SetDurationModal.tsx (hunk 3 above)
Comment (unipadmini) [reply src/nft/components/profile/list/SetDurationModal.tsx:139]: +1

File: src/nft/components/profile/list/SetDurationModal.tsx (hunk 1 above)
Comment (cbachmeier) [reply src/nft/components/profile/list/SetDurationModal.tsx:129]: this was auto-added by lint

File: src/nft/components/profile/list/SetDurationModal.tsx (hunk 2 above)
Comment (cbachmeier) [reply src/nft/components/profile/list/SetDurationModal.tsx:130]: We don't have any plans for future error messages for this dropdown

File: src/nft/components/profile/list/SetDurationModal.tsx (hunk 4 above)
Comment (cbachmeier) [reply src/nft/components/profile/list/SetDurationModal.tsx:95]: No official guidelines, I generally will skip for 1 line, simple executions. If readability is compromised I'll add braces

**Pull request merged**